src/plone/exportimport/cli/__init__.py

```diff
@@ -70,9 +71,9 @@ def importer_cli(args=sys.argv):
         logger.error(f"{namespace.path} does not exist, aborting import.")
         sys.exit(1)
     site = cli_helpers.get_site(app, namespace.site, logger)
-    with api.env.adopt_roles(["Manager"]):
+    with hooks.site(site), api.env.adopt_roles(["Manager"]):
```

Comment:
I did not know you could combine multiple with statements on one line.  Nice!
Apparently it is already [possible since Python 3.1 and 2.7](https://stackoverflow.com/a/1073814/621201).